apps/functions-subscription/src/adapters/azure/functions/get-subscription.ts

```diff
@@ -21,7 +21,11 @@ const makeHandlerKitHandler: H.Handler<
     RTE.apSW(
       'tenant',
       RTE.fromEither(
-        getAndValidateUser(['ApiTrialManager', 'ApiTrialUser'])(req),
+        getAndValidateUser([
+          'ApiTrialManager',
+          'ApiTrialUser',
+          'ApiTrialSupport',
```

Comment:
**issue(blocking)**: the TrialSupport Tenant does not exist.

Here within the handler it's ok to pass the `ApiTrialSupport` parameter, but we also need to let the system know the Tenant `TrialSupport`

Reply:
this means that, without changing the code, the tenant passed to the use case would be an `owner`

Reply:
Added in #261